build.sh

```diff
@@ -188,6 +187,9 @@ if [ "$PIPELINE" = 'build' ]; then
         time docker pull "$GAME_IMAGE" || true
     fi
     date
+    export DOCKER_BUILDKIT=1
+    export STEAM_USERNAME
+    export STEAM_PASSWORD
```

Comment:
ok. better to not have duplicate lines though, maybe move them up to after https://github.com/startersclan/docker-sourceservers/pull/131/files#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823R182

Reply:
For better specificity and security another way is to use:

```sh
DOCKER_BUILDKIT=1 STEAM_USERNAME="$STEAM_USERNAME" STEAM_PASSWORD="$STEAM_PASSWORD" time docker build ...
```

which ensures that no other binary has access to the secrets.

Reply:
ok got to make sure it works

Reply:
```
$ abc1=1 abc2=2 time sh -c 'env|grep abc'
abc1=1
abc2=2
real    0m 0.00s
user    0m 0.00s
sys     0m 0.00s
```